Triangle.java

```diff
@@ -0,0 +1,40 @@
+package pack.homework;
+
+public class Triangle extends Shape{
+    private double sideA;
+    private double sideB;
+    private double sideC;
+
+    public Triangle (double sideA, double sideB, double sideC){
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.sideC = sideC;
+    }
+
+    @Override
+    public double area(){
+        double s = perimeter()/2;
```

Comment:
use `this.perimeter()` instead to mark method as something that is defined in this specific class